docs/index.rst

```diff
@@ -1,53 +1,206 @@
 Ubuntu Pro Client
 #################
 
-The Ubuntu Pro Client (``pro``) is the official tool to enable Canonical
-offerings on your system.
+`Ubuntu Pro`_ is a suite of security services provided by Canonical. Whether
+you’re an enterprise customer deploying systems at scale or want security
+patching for your personal Ubuntu LTS at home, the Ubuntu Pro Client (``pro``)
+is the command-line tool that will help you manage the services you need.
 
-``pro`` provides support to view, enable, and disable the following Canonical
-services:
+The Ubuntu Pro Client comes pre-installed on every Ubuntu system. You can run
+``pro help`` in your terminal window to see a list of the ``pro``
+services and commands available, or get started
+:doc:`with our hands-on tutorial<tutorials/basic_commands>` to try it out in
+a virtual environment.
 
-- `Common Criteria EAL2 (CC EAL) Certification Tooling`_
-- `CIS Benchmark Audit Tooling`_
-- `Ubuntu Security Guide (USG) Tooling`_
-- `Ubuntu Expanded Security Maintenance (ESM)`_
-- `Robot Operating System (ROS) Expanded Security Maintenance`_
-- `FIPS 140-2 Certified Modules (and optional non-certified patches)`_
-- `Livepatch`_
+Which services are for me?
+==========================
 
-If you need any of those services for your machine, ``pro`` is the right tool
-for you. 
+Ubuntu Pro is a broad subscription designed to meet many different needs. You
+are unlikely to want to use all of these tools and services at the same time,
+so you are free to select the precise stream of updates and versions you want
+to apply on any given machine covered by your Pro subscription.
 
-``pro`` is already installed on every Ubuntu system. Try it out by running
-``pro help``!
+To help you navigate the services offered through Ubuntu Pro, click on the tab
+that best describes your needs for our suggestions on which services might be
+of interest to you. For example, Pro includes a set of package versions that
+are compliant with FIPS regulations. You will only want these versions on
+machines that *need* to meet FIPS requirements, so you can choose to enable
+that stream specifically on those machines.
 
------
+.. tab-set::
+
+    .. tab-item:: Commercial
+
+       * **Expanded Security Maintenance (ESM)**
+
+         If you're using an Ubuntu LTS -- Desktop or Server -- in a commercial
+         setting and don't want the inconvenience and downtime of upgrading to
+         a newer LTS yet, you can get additional security support through
+         `Expanded Security Maintenance (ESM)`_. We recommend it for everyone
+         using Pro, and it's enabled by default. It includes two services:
```

Comment:
Ideally we'd work in the value-proposition of esm-apps as well.
```suggestion
         If you're using any Ubuntu LTS -- Desktop or Server -- in a commercial
         setting and you rely on 'universe' packages or if you're on an old LTS but don't want to upgrade
         yet, you can get additional security support through
         `Expanded Security Maintenance (ESM)`_. We recommend it for everyone
         using Pro, and it's enabled by default. It includes two services:
```

Reply:
Good idea! I've changed it to match.